addons/core/translations/form/en-us.yaml

```diff
@@ -119,6 +119,9 @@ to_claim:
 signing_algorithms:
   label: Signing Algorithms
   help: The signing algorithms allowed for the auth method.
+prompts:
+  label: Prompts
+  help: The prompts allowed for an oidc auth method
```

Comment:
none blocking and very picky comment 😅 . I just notive the other `help` values have a . at the end. Can we add that here too?